fdbserver/Status.actor.cpp

```diff
@@ -1242,6 +1242,11 @@ ACTOR static Future<JsonBuilderObject> dataStatusFetcher(WorkerDetails ddWorker,
 				stateSectionObj["name"] = "healthy_removing_server";
 				stateSectionObj["description"] = "Removing storage server";
 			}
+			else if (highestPriority >= PRIORITY_TEAM_HEALTHY) {
```

Comment:
The name and description should match highestPriority >= 0 below:

stateSectionObj["healthy"] = true;
stateSectionObj["name"] = "healthy";
stateSectionObj["name"] = "healthy";